stable/fluent-bit/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluent-bit
-version: 2.4.3
+version: 2.4.5
```

Comment:
its a new functionality so should probably be 2.5.0.

Reply:
bumped version to suggested 2.5.0